pythia_files/hexaquarks_pythia_program.cc

```diff
@@ -101,6 +103,13 @@ int main(int argc, char* argv[]) {
     // save only events containing hexaquarks
     // if(is_hexaquark == false) continue;
 
+    // find jets
+    slowjet.analyze(pythia.event);
+    for(int i = 0; i < slowjet.sizeJet(); i++)
+    {
+      pythia.event.append(99, 1, 0, 0, slowjet.p(i), slowjet.m(i));
```

Comment:
Not sure what the parameters are here, but shouldn't we also set eta and phi?

Reply:
int Event::append(int id, int status, int col, int acol, Vec4 p, double m = 0., double scale = 0., double pol = 9.)
slowjet.p(i) returns a 4-vector of i-th jet